README.md

```diff
@@ -30,10 +30,17 @@ After installing, a `.env` file will be generated which is needed to run huskyCI
 ```sh
 . .env 
 ```
+
+OS X:
```

Comment:
I feel that `OS X` might not be the best, what do you think of using `Mac OS`?

Reply:
Sounds great! Just added `Mac OS`.